google/cloud/forseti/common/util/metadata_server.py

```diff
@@ -64,7 +64,7 @@ def _issue_http_request(method, path, headers):
         http_client.request(method, path, headers=headers)
         return http_client.getresponse()
     except (socket.error, httplib.HTTPException) as e:
-        LOGGER.error('Error with request: %s', e)
+        LOGGER.debug('Error with request: %s', e)
```

Comment:
I think instead we should change METADATA_SERVER_HOSTNAME to the IP instead of the host name, which may not resolve. The IP is '169.254.169.254'. Take a look at https://github.com/GoogleCloudPlatform/google-auth-library-python/blob/master/google/auth/compute_engine/_metadata.py. We may even want to use the environment variable like they are.

Reply:
qq: Is there any concern that the IP address could change in the future?

Reply:
Updated to to use compute engine's metadata ping function to determine whether or not the meta server is reachable